dozer-sql/src/pipeline/expression/python_udf.rs

```diff
@@ -45,18 +45,21 @@ pub fn evaluate_py_udf(
 
         let args = PyTuple::new(py, values);
         let res = function.call1(args)?;
-        Ok(match return_type {
+        if return_type.is_none() {
```

Comment:
Is this the case the return type for python udf is not specified?

Reply:
Yes